v2/setup/fleet.sh

```diff
@@ -9,7 +9,9 @@ fi
 
 # https://gist.github.com/skippy/d539442ada90be06459c
 # TODO: discuss anything else that would be useful here
-AZ=$(curl -s http://169.254.169.254/latest/meta-data/placement/availability-zone)
+if [[ -z $ZONE ]]; then
```

Comment:
It needs to be via CloudConfig. If it isn't, it doesn't break, just hangs on this step until the metadata call times out.